srtcore/api.cpp

```diff
@@ -2827,7 +2843,7 @@ SRTSOCKET CUDT::createGroup(SRT_GROUP_TYPE gt)
     {
         return APIError(e);
     }
-    catch (const std::bad_alloc& e)
+    catch (...)
```

Comment:
Why `bad_alock` was not enough there?

Reply:
There could be some other types of exceptions thrown from the C++ standard library, as I remember. All of them qualify as IPE or at best lethal state of the system resources.